v2/config/rbac/auth_proxy_service.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: v1
-kind: Service
-metadata:
```

Comment:
We should possibly remove the auth_proxy role and rolebinding and just move this into our default role?
```
- apiGroups:
  - authentication.k8s.io
  resources:
  - tokenreviews
  verbs:
  - create
- apiGroups:
  - authorization.k8s.io
  resources:
  - subjectaccessreviews
  verbs:
  - create
```

That's what CAPI did here? https://github.com/kubernetes-sigs/cluster-api/pull/9264/files

Reply:
I'd rather make this `metrics-role` role to keep the things more granular?